cloudformation.yml

```diff
@@ -5,14 +5,30 @@ Parameters:
     Type: String
     NoEcho: true
 
+  HyP3URL:
+    Type: String
+
   EarthdataUsername:
     Type: String
 
   EarthdataPassword:
     Type: String
     NoEcho: true
 
+  S3TargetPrefix:
+    Type: String
+
 Resources:
+  TransferProducts:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      TemplateURL: transfer-products/cloudformation.yml
+      Parameters:
+        HyP3URL: !Ref HyP3URL
+        EarthdataUsername: !Ref EarthdataUsername
+        EarthdataPassword: !Ref EarthdataPassword
+        S3TargetPrefix: !Ref S3TargetPrefix
+
   LogGroup:
```

Comment:
I'd consider creating `hyp3_floods/cloudformation.yml` for symmetry, so each lambda has it's own template, and the top level template just has the two sub-stack resources.

Reply:
and the `hyp3_floods` lambda might benefit from a more explicit name, e.g. `update_flood_subscriptions`